WordPress/src/test/java/org/wordpress/android/ui/jetpack/scan/builders/ScanStateListItemsBuilderTest.kt

```diff
@@ -60,8 +68,8 @@ class ScanStateListItemsBuilderTest : BaseUnitTest() {
             htmlMessageUtils,
             resourceProvider,
             threatItemBuilder,
-            uiHelpers,
-            contextProvider
+            threatDetailsListItemsBuilder,
+            scanStore
         )
 //        whenever(htmlMessageUtils.getHtmlMessageFromStringFormatResId(anyInt(), any())).thenReturn(SpannedString(""))
```

Comment:
❓ Could we remove this commented out code as well? We can always find it in the git history if needed. Wdyt?

Reply:
Yes, removed in d61a3e8.